<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VaultDialog</class>
 <widget class="QDialog" name="VaultDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send Coins</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QFrame" name="frameVaultInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutCoinControl2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutCoinControl" stretch="0,0,0,0,0,0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutCoinControl1">
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Select local file:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="selectButton">
            <property name="toolTip">
             <string>Select document from your computer</string>
            </property>
            <property name="text">
             <string>&amp;Select</string>
            </property>
            <property name="icon">
             <iconset resource="../bitcoin.qrc">
              <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>5</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="infoButton">
            <property name="toolTip">
             <string>Information about Vault feature</string>
            </property>
            <property name="text">
             <string>Info</string>
            </property>
            <property name="icon">
             <iconset resource="../bitcoin.qrc">
              <normaloff>:/icons/transaction_0</normaloff>:/icons/transaction_0</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutCoinControl2" stretch="0,0">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>190</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Document filename:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelDocFile">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widgetCoinControl" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayoutCoinControl5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayoutCoinControl3" stretch="0,0">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="minimumSize">
                <size>
                 <width>190</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>190</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Document SHA512 hash:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelDocHash">
               <property name="text">
                <string/>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutCoinControl4" stretch="0,0">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>190</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="midLineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>Public document name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textDocName">
            <property name="placeholderText">
             <string>Name of the document to publish onto the blockchain</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>190</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Public document version:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textDocVersion">
            <property name="placeholderText">
             <string>Public version of this document, for instance: v1.0.0 (optional)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>190</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Document author:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textDocAuthor">
            <property name="placeholderText">
             <string>Author name and contact information in any format (optional)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>190</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Public comments:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textComment">
            <property name="placeholderText">
             <string>Additional information about this document and its copyright (optional)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="toolTip">
        <string>Clear form</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="publishButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Confirm the document publish action</string>
       </property>
       <property name="text">
        <string>&amp;Publish</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/send</normaloff>:/icons/send</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
